lib/degreed/content/courses.rb

```diff
@@ -52,6 +56,43 @@ def all(external_id: nil)
         request.get(content_courses_url, params: params)
       end
 
+      # Update course
+      #
+      # @see https://api.degreed.com/docs/#update-a-specific-course
+      #
+      # @param title [String]
+      # @param external_id [String, #to_s]
+      # @param url [String]
+      # @param duration [Integer, #to_i]
+      # @param duration_type [String]
+      #   Accepted values: Seconds, Minutes, Hours, or Days
+      # @param summary [String] Optional (default: nil)
+      #
+      # @return [Degreed::Response]
+      def update(id:, **kwargs)
```

Comment:
Two things I really like about your approach here:
- The id field is a keyword argument without a default, enforcing that it's required. If they don't pass an ID, Ruby will give them a helpful error message saying that key is required.
- I really like that we are making it possible to supply arguments with underscore, which is more idiomatic Ruby, rather than "dasherized" 🎉